### PR TITLE
Ana/3977 link success modal to activity

### LINKED ISSUE #3977: kusama integration feedback
> When I try to login, I need to agree that it’s a testnet address. Kusama shouldn’t be referred to as a testnet, see the following article https://medium.com/polkadot-network/kusama-polkadot-comparing-the-cousins-170e4fe6c280.

> “Waiting for confirmation from polkadot-testnet” should be changed. It’s not a testnet see above ;-).

i guess we'll just call it a mainnet of sorts. let's switch the chain-id to `kusama` or `polkadot-canary` instead of `polkadot-testnet`?

> On Kusama it only takes 7 days to unlock your tokens and not 21 (notification when I stake)

simple fix. let's update the language. ideally, we would pull this from somewhere instead of hardcoding — but for now let's just adjust the message assuming cosmos is 21 and polka/kusama is 7.

> When I click on stake or sending tokens it doesn’t show any transaction fee, but it removes the fee.

bug?

> Staking doesn’t seem to work. “No validators in your portfolio” 

bug again?

> KSM currently has no currency on android, even though it let’s you select Euro, Dollar, etc.

does coingecko have KSM support?

> I think a 1) transaction history or an 2) address book would be super useful.

1) let's change the message on the success screen in the action modal to link to the activity page instead of the block page.

2) one day @michielmulders 

> The recovery feature seems to be loading the address forever and if I just continue with the password and click on create it’s say unreachable.

bug again?

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3977
**Description:**

I think with this one we should have covered all their feedback remarks. Validators are showing for me when I stake and transactions are going through with fees being displayed.

We're good to go!

This is how it looks now:

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
what happens if the transactions page isn't implemented yet?

**Comment by jbibla** [comment]
nice! can you shorten the address `cosmos...1234` like we usually do?

**Comment by Bitcoinera** [comment]
> what happens if the transactions page isn't implemented yet?

You just see the Activity page, just like with push notifications

**Comment by Bitcoinera** [comment]
> nice! can you shorten the address `cosmos...1234` like we usually do?

This is what I got for now... I don't know why the `Address` component is not rendering at all 👀

**Comment by faboweb** [comment]
unit tests are failing

**Comment by jbibla** [comment]
> You just see the Activity page

yeah but we don't have this implemented for kusama yet... maybe if we don't have transaction history implemented it should just go back to profile page?

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -191,12 +191,12 @@
             <div slot="subtitle">
               {{ notifyMessage.body }}
               <br />
-              <br />Block
+              <br />See your transaction
```
Comment (jbibla) [inline src/ActionModal/components/ActionModal.vue:194]: sorry to be annoying - but i think this whole thing should be a link "See your transaction" and we don't need the word "here".

Comment (Bitcoinera) [reply src/ActionModal/components/ActionModal.vue:194]: oh ok, makes sense. Let's change it

**Comment by Bitcoinera** [comment]
> > You just see the Activity page
> 
> yeah but we don't have this implemented for kusama yet... maybe if we don't have transaction history implemented it should just go back to profile page?

Maybe link to Polkascan for now? 😅 or add a "Coming Soon"?

**Comment by faboweb** [comment]
@jbibla can you clarify?

**Comment by jbibla** [comment]
nevermind — if ana can change the link we should merge this!

**Comment by Bitcoinera** [comment]
> nevermind — if ana can change the link we should merge this!

Done!

**Pull request merged**